oxo.py

```diff
@@ -1,26 +1,81 @@
 class OxoBoard:
+    empty = 0
```

Comment:
Good use of a named constant -- why not also define them for players `O` and `X`?